### PR TITLE
feat(mobile-score): Add mobile profile to relay config

### PR DESCRIPTION
**Pull request by narsaynorath**
Adds the mobile profile that we're treating as an alpha version. This profile is just to ensure that we're able to ingest the values we want and we will iterate the profiles in the future.

The browser configs are extracted into a helper function with no modifications, and there is a mobile profile added. The `config.performanceScore.profiles` path will only be added if either of these helpers populates an array with profiles to pass along.

⚠️ Do not merge until https://github.com/getsentry/relay/pull/3249 is merged

### CODE REVIEW

File: src/sentry/relay/config/__init__.py [hunk 1]
```diff
@@ -489,300 +845,12 @@ def _get_project_config(
...
-                    "condition": {
-                        "op": "eq",
-                        "name": "event.contexts.browser.name",
-                        "value": "Opera",
-                    },
-                },
-            ]
-        }
+    config["performanceScore"] = {
+        "profiles": [
```
Comment (iker-barriocanal) [inline src/sentry/relay/config/__init__.py:849]: Previously, orgs without the feature flag enabled would not have `config.performanceScore.profiles` in the config. Could this still be the case?

**Review by iker-barriocanal — CHANGES_REQUESTED** [review]
Could we note the parts of the config that are changing? For example, comments in the diff or in the PR description. It's hard to track what's changing in the current diff.

File: src/sentry/relay/config/__init__.py (hunk 1 above)
Comment (narsaynorath) [reply src/sentry/relay/config/__init__.py]: I've added a condition to only apply these properties to the config if we have any profiles to add. Essentially I've just written helpers that use the feature flags internally to decide which/if there are profiles

**Review by iker-barriocanal — APPROVED** [review]
Thanks!

**Review by edwardgou-sentry — APPROVED** [review]

**Pull request merged**